### ISSUE TITLE
csg_stat: Topology does not have beads of type "ME"

### ISSUE THREAD

**Issue by junghans** [votca/csg#302]
After merging #298, I get the following error in when running the `methanol-water_X_0.062_simplex` repression test:
```
csg_stat --nt 4 --options /home/christoph/votca/src/votca/build/csg-tutorials/methanol-water/X_0.062/re/settings.xml --top topol.tpr --trj traj.xtc --begin 0 --first-frame 0
begin to calculate distribution functions
# of bonded interactions: 0
# of non-bonded interactions: 3
Reading file topol.tpr, VERSION 2019-rc1-dev-20181217-b3b270485-dirty-unknown (double precision)
I have 4000 beads in 4000 molecules
Reading file topol.tpr, VERSION 2019-rc1-dev-20181217-b3b270485-dirty-unknown (double precision)
Reading file topol.tpr, VERSION 2019-rc1-dev-20181217-b3b270485-dirty-unknown (double precision)
Reading file topol.tpr, VERSION 2019-rc1-dev-20181217-b3b270485-dirty-unknown (double precision)
Reading frame       0 time    0.000
an error occurred:
Topology does not have beads of type "ME"
This was specified in type1 of interaction "ME-ME"
```

**Comment by JoshuaSBrown** [comment]
Um, how do you create the .xtc file?

**Comment by junghans** [comment]
Just use `conf.gro` instead of `traj.xtc`. `topol.tpr` can be created by running `gmx grompp`.

**Comment by junghans** [comment]
Fixed in #305
